styles/facts.css

```diff
@@ -0,0 +1,58 @@
+/* add background image */
+body {
+  background-image: url('https://cdn.pixabay.com/photo/2020/05/17/04/22/pizza-5179939_960_720.jpg');
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+
+h3 {
+    color:rgb(247, 156, 171);
+    font-family: fantasy;
+    font-size: 18px;
+    font-weight: bold;
+}
+
+
+p {
+    font-family: 'Courier New', Courier, monospace;
+}
+
+
+/* Add Grid */
+.grid-container {
+  display: grid;
+  padding: 40px;
+  gap: 20px;
+  grid-template-columns: 1fr 1fr;
+  justify-items: center; 
+  align-items: center;
+}
+
+
+.fun-facts {
+  background-color: lightblue;
+  width: 520px;
+  height: 200px;
+  padding: 30px;
+  display: inline-block;
+  border: 2px solid rgb(247, 156, 171);
+
```

Comment:
For consistency and avoiding confusion (like developers wondering if there was a deleted rule that should be there) we should remove the blank lines inside rule sets.